packages/core/src/env-set/oidc.ts

```diff
@@ -87,18 +77,20 @@ export const readPrivateKeys = async (): Promise<string[]> => {
   const notExistPrivateKeys = privateKeyPaths.filter((path): boolean => !existsSync(path));
 
   if (notExistPrivateKeys.length > 0) {
-    const notExistPrivateKeysRawValue = JSON.stringify(notExistPrivateKeys);
     throw new Error(
-      `Private keys ${notExistPrivateKeysRawValue} configured in env \`OIDC_PRIVATE_KEY_PATHS\` not found.`
+      `Private keys ${listFormatter.format(
+        notExistPrivateKeys
+      )} configured in env \`OIDC_PRIVATE_KEY_PATHS\` not found.`
     );
   }
 
   try {
     return privateKeyPaths.map((path): string => readFileSync(path, 'utf8'));
   } catch {
-    const privateKeyPathsRawValue = JSON.stringify(privateKeyPaths);
     throw new Error(
-      `Failed to read private keys from ${privateKeyPathsRawValue} in env \`OIDC_PRIVATE_KEY_PATHS\`.`
+      `Failed to read private keys ${listFormatter.format(
+        ' and '
+      )} from env \`OIDC_PRIVATE_KEY_PATHS\`.`
```

Comment:
These two error messages can be aligned

Reply:
They're not the same.
The first is a message for the missing file.
The second one is a failure when reading the keys.

Reply:
Yes, I get it. They load different things. But what I'm saying is to align the wording of the error messages. Use both `Failed to read blah blah and blah` for example

Reply:
Removed, for we don't need to handle this exception.